### DISCUSSION TITLE
All opened scenes show in 2D view of one scene, bug or feature?

### DISCUSSION THREAD

**Discussion by hsandt** [godotengine/godot-proposals#6993] [2D]
I recently noticed that the 2D view showed all my open scenes at once:

![2023-05-31 Godot 4 0 2 Opened tutorial scene still shows other scenes](https://github.com/godotengine/godot-proposals/assets/1067772/8a20ba3e-5e5f-46f9-b8e9-11feed0b26e4)

after closing all other scene tabs, this is what I get (the expected behavior):

![image](https://github.com/godotengine/godot-proposals/assets/1067772/05bbe8f6-c0b8-48a3-858a-8d99681c2a44)

After reopening another scene though, I cannot repro.

It looks like a bug. But at the same time, seeing multiple scenes at once to preview a certain scene in the context of another one sounds like a cool feature to have. So maybe I clicked some option "show as context is all scene views" by accident?

If it's a feature, what option did I enable to trigger that? And then my proposal would be to make this option more obvious so I can disable it.

If it's a bug though, I cannot repro, but not sure if worth opening a bug report...
I suppose it _could_ also become a proposal as I find editing "prefab" scenes in context quite convenient. But if the bug is triggered by some obscure circumstances, it's probably not worth reexploiting that code to actually show scenes in context, and we may as well restart work on this from scratch.

**Comment by Zireael07** [comment]
Looks like a bug - what's the Godot version and your GPU?

**Reply by hsandt** [reply]
Godot Engine v4.0.2.stable.official.7a0977ce2 - https://godotengine.org
OpenGL API 3.3.0 NVIDIA 525.105.17 - Compatibility - Using Device: NVIDIA Corporation - NVIDIA GeForce GTX 860M

**Comment by digilena** [comment]
I am also facing this in the following version:
Godot Engine v4.2.1.stable.official.b09f793f5 - https://godotengine.org
OpenGL API 4.1 Metal - 88 - Compatibility - Using Device: Apple - Apple M2

**Reply by hsandt** [reply]
oh, that's a much more recent repro! I didn't have it myself again but that's great if you can repro and video capture / copy log output (in this case you can probably open a bug report instead since it seems to be a bug indeed)

**Comment by Calinou** [comment]
Ideally, we'd need a video recording of your Godot editor session from when you start the editor until you encounter the issue to be able to troubleshoot this. I've never been able to reproduce this on my end.

**Reply by hsandt** [reply]
this project too old for me and it didn't happen again the few times I went back to it to add some stuff, sorry...

**Reply by AThousandShips** [reply]
I'd suggest closing this if you can't replicate it, and if you come across it again opening a normal bug report
